test/hybig/HyBIG_Regression.ipynb

```diff
@@ -824,6 +834,300 @@
     "else:\n",
     "    print('Skipping test: HyBIG regression tests not configured for this environment.')"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c567bc0e",
+   "metadata": {},
+   "source": [
+    "### Test that demonstrates variable selection for custom colour maps:\n",
+    "\n",
+    "This test will run a request that specifies a variable for HyBIG to use. It is a bit of a quirky way to ensure HyBIG has access to the related URL for a custom colour map. The output should be in colour rather than greyscale.\n",
+    "\n",
+    "PREFIRE granules also cover the whole Earth at sufficient resolution such that HyBIG will create tiled output, so there will be 15 tiles generated (3 rows, 5 columns).\n",
+    "\n",
+    "This test primarily seeks to ensure the values in the output images use the colour map specified in the related URLs of the specified [UMM-Var record](https://cmr.uat.earthdata.nasa.gov/search/variables.umm_json?concept_id=V1270130943-LARC_CLOUD)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "879897b6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "prefire_basename = 'PREFIRE_SAT2_2B-FLX_S07_R00_20210721013413_03040.nc.G00'\n",
+    "\n",
+    "# Define the tile information, so the output for each tile can be tested:\n",
```

Comment:
I probably could reduce this by a spatial region, rather than processing all tiles, but they are all small files, so the biggest thing is just that this notebook cell is a bit long and repetitive.